core/api/src/graphql/admin/schema.graphql

```diff
@@ -424,6 +429,11 @@ type UsdWallet implements Wallet {
 
   """An unconfirmed incoming onchain balance."""
   pendingIncomingBalance: SignedAmount!
+  pendingTransactions: [Transaction!]!
+  pendingTransactionsByAddress(
```

Comment:
how is this working if you need to pass wallet ids? did you test it?

Reply:
You dont need to pass wallet ids, this is already on the wallet object, so it uses the id of the wallet it is on.